_overviews/jdk-compatibility/overview.md

```diff
@@ -4,70 +4,79 @@ title: JDK Compatibility
+## Long Term Support (LTS) versions
 
-Scala code compiled on Java 8 should run without problems in later JVMs, and we will give higher priority to bugs that break this property. For example, in the 2.13.x series we intend to provide support for JPMS module access checks, to allow ensuring your code won't incur `LinkageErrors` due to module access violations.
+After Java 8, Oracle introducd the concept of LTS versions of the JDK. These versions will remain supported (by Oracle, and likely by the rest of the ecosystem, including Scala) for longer than the versions in between. See <https://www.oracle.com/technetwork/java/eol-135779.html>.
 
-## Dropping old JVMs
+JDK 8 and 11 are LTS versions. The next LTS version will be JDK 17, planned for September 2021.
 
-Some Scala releases increase the *required* JVM versions for a Scala release. To leverage new features offered by a JVM release, we must sometimes drop support for older JVMs.
+Scala provides experimental support for running the Scala compiler on non-LTS versions of the JDK. The current LTS versions are normally tested in our CI matrix and by the Scala community build. We may also test non-LTS versions, but any issues found there are considered lower priority, and will not be considered release blockers. (Lightbend may be able to offer faster resolution of issues like this under commercial support.)
 
-For example, Scala 2.12 raised the minimum JVM, for both compiling and running, from version from 6 to 8. This was done so we could take advantage of new features in 8 such as lambdas and default methods.
+As already mentioned, Scala code compiled on JDK 8 should run without problems in later JVMs. We will give higher priority to bugs that break this property. (For example, in the 2.13.x series we intend to provide support for JPMS module access checks, to ensure your code won't incur `LinkageErrors` due to module access violations.)
 
-Like the 2.12.x series, the Scala 2.13.x series will support Java 8 and higher. (We may bump this to Java 11, but this is unlikely to happen in the 2.x series.)
+## JDK vendors and distributions
+
+We build and test Scala primarily on AdoptOpenJDK, but you're free to use the JDK and JVM of your choice.
+
+JDK 8 users typically use the Oracle JDK or some flavor of OpenJDK.
+
+Most JDK 11+ users are using either OpenJDK or GraalVM.
+
+OpenJDK comes in various flavors, offered by different providers.  We build and test Scala using AdoptOpenJDK in particular, but the differences are unlikely to matter to most users.
```

Comment:
"We build and test Scala using AdoptOpenJDK" is a duplicate with the first sentence of this section.

Reply:
fixed